src/content/reference/react/captureOwnerStack.md

```diff
@@ -0,0 +1,301 @@
+---
+title: captureOwnerStack
+---
+
+<Wip>
+
+**This API is experimental and is not available in a stable version of React yet.**
+
+You can try it by upgrading React packages to the most recent experimental version:
+
+- `react@experimental`
+
+Experimental versions of React may contain bugs. Don't use them in production.
+
+</Wip>
+
+<Intro>
+
+`captureOwnerStack` reads the current Owner Stack in development and returns it as a string if available.
+
+```js
+const stack = captureOwnerStack();
+```
+
+</Intro>
+
+<InlineToc />
+
+---
+
+## Reference {/*reference*/}
+
+### `captureOwnerStack()` {/*captureownerstack*/}
+
+Call `captureOwnerStack` to get the current Owner Stack.
+
+```js {5,5}
+import {captureOwnerStack} from 'react';
+
+function Component() {
+  if (process.env.NODE_ENV !== 'production') {
+    const ownerStack = captureOwnerStack();
+    console.log(ownerStack);
+  }
+}
+```
+
+#### Parameters {/*parameters*/}
+
+`captureOwnerStack` does not take any parameters.
+
+#### Returns {/*returns*/}
+
+`captureOwnerStack` returns `string | null`.
+
+If no Owner Stack is available, it returns an empty string.
+Outside of development builds, `null` is returned.
```

Comment:
```suggestion
If no Owner Stack is available, it returns an empty string. Outside of development builds, `null` is returned.
```

Also, instead of "if no owner stack is available" we should say when owner stacks are not available. Like maybe not too specific, but something like "if it's not called inside a component lifecycle".

Reply:
We call this out in the troubleshooting section so we should just also include that explanation here.

Reply:
component lifecycle can read too specific. It's not obvious that `onCaughtError` is part of the component lifecycle. 

I'll be more specific here though.